### PR TITLE
my contribution

### PR DESCRIPTION
**Pull request by monstercloud**
merge pls!

### CODE REVIEW

File: new-file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+this is a bloody new file
```
Comment (irsl) [inline new-file.txt]: this fiel is completely pointless please remove

File: README.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
 # test1
 test1
+
+some modificaiton
```
Comment (irsl) [inline README.md]: the original was fine, remove.

File: new-file.txt (hunk 1 above)
Comment (monstercloud) [reply new-file.txt]: removed as requested

File: README.md (hunk 2 above)
Comment (monstercloud) [reply README.md]: removed line as requested

**Pull request merged**